src/component/Header/Header.tsx

```diff
@@ -0,0 +1,49 @@
+import React, {FC} from 'react';
+import './Header.scss';

```

Comment:
React is case sensitive and we have created header.scss and we are trying to import Header.scss I think it'll occur an error.

Reply:
Got it.. issue has been resolved